Examples/StateReconstruction/ising_1d_qsr.py

```diff
@@ -0,0 +1,95 @@
+import netket as nk
+import numpy as np
+from netket.driver import QSR
```

Comment:
Hi @JasonZHM, thanks for the contribution! One minor thing: Typically, main NetKet drivers are exposed directly under the `netket` top-level module for convenience (i.e., as `netket.QSR` in this case).

There is no need to move the existing code, you can just add an import here (and update the example):
https://github.com/netket/netket/blob/a916167944ba6df7394b6a7274fbfdb5c6550a4b/netket/__init__.py#L61-L63